docs/internal/config/List-Core-Actions.md

```diff
@@ -1,9 +1,15 @@
-# List of Core Actions
-
-The snapshot loaded with the system at the start, called `core_templates`, has a template called `Sample template`, which contains all the core actions listed here.
-
+## Contents <!-- omit in toc -->
+<!-- toc -->
+- [Disclaimer](#disclaimer)
+- [Briefing](#briefing)
+- [Lists](#lists)
+  - [Templates](#templates)
+  - [Core-actions](#core-actions)
+  - [Other actions](#other-actions)
+<!-- tocstop -->
 ## Disclaimer
 
+The snapshot loaded with the system at the start, called `core_templates`, has a template called `Sample template`, which contains all the core actions listed here.  
 The plugin actions that will be listed here are different from actions that users can act in the sytem, this are only meant for admins and configure how the system will work.
```

Comment:
There's duplication here of the "core action" list (and it's out of date). Can you just replace that section with a link to https://github.com/openmsupply/conforma-server/wiki/List-of-Action-plugins#core-actions ?

Reply:
Yes - for sure! Thanks for suggesting